### PR TITLE
DOC: Explain the term "image registration" in README

### PR DESCRIPTION
**Pull request by N-Dekker**
Addresses issue https://github.com/SuperElastix/elastix/issues/757 "Please add an explanation in layman terms for the term "image registration" to the README", submitted by yuri@FreeBSD (@yurivict).

Added text based upon the manual: https://github.com/SuperElastix/manual/blob/779e9503f7f1b4ad658d361486d5d4f0e6c7baa2/manual.tex#L180-L182

### CODE REVIEW

**Comment by mstaring** [comment]
Nice improvement! "finding a spatial one-to-one mapping" is still quite technical though. How about something like "finding the overlay between two images taken at different time points or orientations"

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 Welcome to elastix: a toolbox for rigid and nonrigid registration of images.
 
-elastix is open source software, based on the well-known [Insight Segmentation and Registration Toolkit (ITK)](https://itk.org/). The software consists of a collection of algorithms that are commonly used to solve (medical) image registration problems. The modular design of elastix allows the user to quickly configure, test, and compare different registration methods for a specific application. A command-line interface enables automated processing of large numbers of data sets, by means of scripting.
+elastix is open source software, based on the well-known [Insight Segmentation and Registration Toolkit (ITK)](https://itk.org/). The software consists of a collection of algorithms that are commonly used to perform (medical) image registration: the task of finding a spatial one-to-one mapping from voxels in one image to voxels in the other image. The modular design of elastix allows the user to quickly configure, test, and compare different registration methods for a specific application. A command-line interface enables automated processing of large numbers of data sets, by means of scripting.
```
Comment (N-Dekker) [inline README.md:15]: Or otherwise like this? "the task of finding a spatial transformation, mapping one image (the _fixed_ image) to another (the _moving_ image), by optimizing some applicable image similarity metrics"

Comment (N-Dekker) [reply README.md:15]: > "finding a spatial one-to-one mapping" is still quite technical though. How about something like "finding the overlay between two images taken at different time points or orientations"?

@mstaring Personally I'm in favor of keeping it a little bit technical, or at least concise and correct. This is just the README file, it should just shortly specify what elastix is about, in my opinion. An extensive explanation for laymen on what image registration is all about could of course be added elsewhere, maybe in the manual, if necessary.

Comment (mstaring) [reply README.md:15]: ok, agreed

Comment (N-Dekker) [reply README.md:15]: Thanks @mstaring I think I'll go for "the task of finding a spatial transformation, mapping one image (the _fixed_ image) to another (the _moving_ image), by optimizing some applicable image similarity metrics" https://github.com/SuperElastix/elastix/pull/774#discussion_r1054409168 because it already refers to those fundamental concepts of elastix, _fixed/moving images, transformations, optimizations,_ and _metrics_.
